v23.1/full-text-search.md

```diff
@@ -0,0 +1,472 @@
+---
+title: Full-Text Search
+summary: Full-text searches using TSVECTOR and TSQUERY enable natural-language searches on documents with ranked results.
+toc: true
+docs_area: develop
+---
+
+A full-text search is used to perform natural-language searches on documents such as articles, websites, or other written formats.
+
+This page describes how to perform full-text searches using the provided [built-in functions](functions-and-operators.html#full-text-search-functions).
+
+{{site.data.alerts.callout_info}}
```

Comment:
Nit: "currently" technically this goes against the style guide... though maybe there is a need to add this here?

Reply:
Oh, I did not realize. Removing.